SConstruct

```diff
@@ -277,18 +279,21 @@ elif env['CC'] == 'cl': # Visual Studio
     defaults.buildPch = True
     env['pch_flags'] = ['/FIpch/system.h']
     env['openmp_flag'] = '/openmp'
+    defaults.includeRPath = False
```

Comment:
Do we need to special case this, or will SCons just ignore this if it's not an option for Visual Studio?

Reply:
It looks like you're right, [SCons ignores it for windows](https://scons.org/doc/2.3.6/HTML/scons-man.html#cv-RPATH)